### PR TITLE
Use PyPi dependencies for running the scrapers and tests

### PR DESCRIPTION
**Pull request by mprahl**
This also pins the version of neomodel to avoid pulling in pytest
and its dependencies in production. This should be fixed by:
https://github.com/neo4j-contrib/neomodel/pull/553

This will enable the base image to be changed to UBI8.

### CODE REVIEW

**Comment by mprahl** [comment]
The flake8 error will go away after it is rebased on top of #301

**Review by apaplaus — APPROVED** [review]

**Comment by apaplaus** [comment]
I forgot to change the installation of rpms in other Dockerfiles, so thanks for fixing it.

File: README.md [hunk 1]
```diff
@@ -35,9 +35,8 @@ Or
...
 
-The unpinned dependencies are recorded in **setup.py**, and to generate the **requirements.txt** file, run `pip-compile --generate-hashes --output-file=requirements.txt --allow-unsafe`. This is only necessary when adding a new package. To upgrade a package, use the `-P` argument of the `pip-compile` command.
-
-To update **docs-requirements.txt** or **scraper-requirements.txt**, run `pip-compile --generate-hashes docs-requirements.in -o docs-requirements.txt --allow-unsafe` or `pip-compile --generate-hashes scraper-requirements.in -o scraper-requirements.txt --allow-unsafe` respectively. Also, there are test dependencies specified in **tests/requirements.txt** to regenerate this file run `pip-compile --generate-hashes tests/requirements.in -o tests/requirements.txt --allow-unsafe`. All those commands are meant to be run from the root directory of the project.
+The unpinned dependencies are recorded in **setup.py**, and to generate the **requirements.txt** file, run `make pin_dependencies`. This is only necessary when adding a new
```
Comment (lioramilbaum) [inline README.md:38]: It is necessary when requirements.in file is changed: adding/removing a package.

**Comment by lioramilbaum** [comment]
I am reviewing the PR although it consists of multiple changes. It is challenging to provide an effective review in a timely manner.

**Review by lioramilbaum — APPROVED** [review]

**Review by lioramilbaum — CHANGES_REQUESTED** [review]
CI is broken

**Comment by apaplaus** [comment]
> CI is broken

Yes, it's an error cause by flake8. Matt mentioned here(https://github.com/redhat-exd-rebuilds/estuary-api/pull/303#issuecomment-819643068) that the error will be gone after merging on top of his other change.

**Comment by lioramilbaum** [comment]
> > CI is broken
> 
> Yes, it's an error cause by flake8. Matt mentioned here([#303 (comment)](https://github.com/redhat-exd-rebuilds/estuary-api/pull/303#issuecomment-819643068)) that the error will be gone after merging on top of his other change.

ok. Let's wait then till the other PR is merged and this PR is rebased, before we move on and merge this one.

**Pull request merged**